modules/rte_mis_mail/src/Form/EmailSettingsForm.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\rte_mis_mail\Form;
+
+use Doctrine\Common\Annotations\Annotation\Required;
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\mobile_number\MobileNumberUtilInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Provides a form to configure Mobile Number settings.
+ */
+class EmailSettingsForm extends ConfigFormBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'custom_email_validation_settings_form';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function getEditableConfigNames() {
+    return ['custom_email_validate.settings'];
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $config = $this->config('custom_email_validate.settings');
```

Comment:
email_form.settings.

Reply:
Combined the forms and modified to email_sms_form.settings.